config/sync/core.base_field_override.node.campaign.promote.yml

```diff
@@ -0,0 +1,22 @@
+uuid: d0aef694-fbaf-44ff-b959-309295a08dd9
+langcode: da
+status: true
+dependencies:
+  config:
+    - node.type.campaign
+id: node.campaign.promote
+field_name: promote
+entity_type: node
+bundle: campaign
+label: 'Forfremmet til forside'
```

Comment:
Consider whether the new content type should have a promoted property.

I do not think it makes sense in the current context and should be left out if possible. On the other hand a the standard published property would make sense.

Reply:
Allthough that this configuration is generated the UI behaves like the functionality has been deactivated and I don't see that I have more options than I already have been configuring:

Configuration of the content type:
<img width="956" alt="image" src="https://user-images.githubusercontent.com/998889/194851828-3ca72664-25d3-4665-8427-0e417702ab6a.png">

Configuration of the edit formular:
<img width="997" alt="image" src="https://user-images.githubusercontent.com/998889/194851907-b3dcee0f-3a2e-4819-8e8e-79ede8c1306f.png">